cypress/features/regression/dialogClassic.feature

```diff
@@ -4,7 +4,16 @@ Feature: Dialog component classic story
   Background: Open Dialog component page classic story
     Given I open "Dialog" component page classic
 
+  @positive
+  Scenario: CloseIcon has the border outline
+    When I open component preview
+    Given I uncheck showCloseIcon checkbox
+    When I check showCloseIcon checkbox
+    Then closeIcon is visible
+    Given closeIcon is focused
+    Then closeIcon has border outline for classic story
```

Comment:
```
Scenario: CloseIcon has the border outline
     When closeIcon is focused
     Then closeIcon has the border outline

Reply:
I put in slightly different implementation, fixed: ffc3c28